hw/ip/otbn/doc/_index.md

```diff
@@ -516,6 +516,9 @@ The data memory (DMEM) is 256b wide and read-write accessible from the base and
 When accessed from the base instruction subset through the `LW` or `SW` instructions, accesses must read or write 32b-aligned 32b words.
 When accessed from the big number instruction subset through the `BN.LID` or `BN.SID` instructions, accesses must read or write 256b-aligned 256b words.
 
+Both memories can be accessed through OTBN's register interface ({{< regref "DMEM" >}} and {{< regref "IMEM" >}}) while OTBN is idle (as indicated by {{< regref "STATUS.busy">}}).
```

Comment:
Sounds good. Should we explicitly note that this will fail when OTBN is busy? (Or do we say that somewhere else?)

Reply:
What kind of failure? a bus hang or a bus error?

Reply:
I added a "only" to the text. We do (intend to) specify the exact behavior in the register description. We currently say there "          Not accessible during the operation of the engine.           TODO: Discuss and document behavior in that case. Alert? Ignore?" This is something we need to discuss further at some point, it's tracked in https://github.com/lowRISC/opentitan/issues/2696.